src/model/stream.ts

```diff
@@ -0,0 +1,40 @@
+import { Offset } from "src/gen/model/v1/model";
```

Comment:
The same. Is it copied from `stream-registry-api`? Let's try to reuse that code. Duplicates lead to maintainability problems because modifying one of the copies we have to remember to modify other ones.